theming.js

```diff
@@ -165,16 +165,21 @@ var ThemeManager = class DashToDock_ThemeManager {
             if (settings.get_enum('transparency-mode') == TransparencyMode.FIXED)
                 newAlpha = settings.get_double('background-opacity');
 
-            backgroundColor = Clutter.color_from_string(settings.get_string('background-color'))[1];
```

Comment:
This is where _part_ of the issue with background colors is occuring. If I recall correctly there was some odd things happening with GTK/GDK4's default formatting (from prefs)